v20.2/performance.md

```diff
@@ -62,6 +70,8 @@ For benchmarking latency, again, Cockroach Labs believes TPC-C provides the most
 
 For example, when running Sysbench on a 3-node cluster of AWS `c5d.9xlarge` machines across AWS’s `us-east-1` region (availability zones `a`, `b`, and `c`), CockroachDB can achieve an average of 4.3ms on the `oltp_insert` workload and 0.7ms on the `oltp_point_select` workload.
 
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update this chart with latest data - see <https://github.com/cockroachdb/docs/issues/8212>
+
 <img src="{{ 'images/v20.2/sysbench-latency.png' | relative_url }}" alt="Sysbench Latency" style="max-width:100%" />
```

Comment:
Same questions/comments as above but for this Sysbench latency chart:

- Is there some updated data for these Sysbench runs that can be used to update this chart?
- I can take a stab at updating the chart but I'm not sure where / how to get the latest data.